src/lib/divider/divider.scss

```diff
@@ -0,0 +1,66 @@
+.mat-divider {
+  display: block;
+  margin: 0;
+  border-top-width: 1px;
```

Comment:
For the dividers line width (based on the border width), can you move the value (`1px`) into a variable at the top of the file?

Reply:
Done